src/components/appheader.main.less

```diff
@@ -113,50 +127,75 @@
 
         .icon {
           color: #fff;
-          font-size: 20px;
+          font-size: 15px;
+        }
+
+        .hamburger-shop{
+          font-size: 7px;
         }
       }
     }
   }
 
+  .cart-link-counter {
+    text-align: center;
+    position: absolute;
+    background-color: @mainBlueColor;
+    min-width: 16px;
+    height: 16px;
+    font-size: 12px;
+    line-height: 16px;
+    border-radius: 50%;
+    color: white;
+    top: 0;
+    left: 0;
+    font-weight: bold;
+  }
+
   .collapsable-container {
     display: none;
   }
 
   @media (max-width: @tabletWidth - 1px) {
     .main-container {
+      justify-content: flex-end;
       .back-btn-container {
         display: block;
       }
 
       .logo-container {
         padding-left: 0px;
         padding-top: 0px;
+        display: block;
+        position: absolute;
+        left: 50%;
+        transform: translateX(-50%);
       }
 
       .central-container {
-        .horizontal-menu {
-          display: none;
-        }
+        display: none;
       }
 
       .search-container {
-        display: none;
+        display: inline-block;
```

Comment:
Hmm I noticed this now will overlap with the logo on a search. 
![image](https://user-images.githubusercontent.com/39800563/50233642-dc1c0880-0381-11e9-8870-3d154b6ed474.png)

I proposed something to the designer based on this and waiting on feedback from them. I'm proposing the `search` button click triggering the dropdown to expand, to reveal the search form in the dropdown:
![image](https://user-images.githubusercontent.com/39800563/50233677-f1913280-0381-11e9-92f9-2cc473847f54.png)

Reply:
Thank Shaun for pointing out, we will fix this issue tomorrow morning.

Reply:
Hey @rostyk-kanafotskyy! I got a reply back from the designer and the alternative is approved for implementation. The search button click can trigger the dropdown to expand, then put focus on the form field in the dropdown. Thank you!